theme_report/generate_theme_report.py

```diff
@@ -1,31 +1,25 @@
-from selenium import webdriver
-from selenium.webdriver.common.by import By
-from selenium.webdriver.support.ui import WebDriverWait  # available since 2.4.0
-# available since 2.26.0
-from selenium.webdriver.support import expected_conditions as EC
+from playwright.sync_api import Playwright, sync_playwright
+
 from bs4 import BeautifulSoup
 
 from canvasapi import Canvas
 import pandas as pd
+
 import os
 import json
 import logging
+import time
 
-logger = logging.getLogger(__name__)
+import sys
 
+logging.basicConfig(level=logging.INFO)  # Set the logging level to INFO
 
-def parse_subaccount_theme(account, driver, API_URL, report_df):
+def parse_subaccount_theme(account, page, API_URL, report_df):
     """
     parse html of subaccount theme page
     """
-    print(f"parse subaccount {account}")
-    driver.get(f"{API_URL}/accounts/{account.id}/brand_configs")
-    element = WebDriverWait(driver, 30).until(
-        EC.presence_of_element_located((By.ID, "left-side")))
-    element = driver.find_element_by_xpath('//*')
-    innerHtml = element.get_attribute('innerHTML')
-    soup = BeautifulSoup(innerHtml, features="html.parser")
-
+    page.goto(f"{API_URL}/accounts/{account.id}/brand_configs")
```

Comment:
I've found that after a certain number of page redirects, my Chromium window closes after this line without any trace of an error or exception, leaving the script hanging. When editing the code to only run one account there's no problem, but the whole list typically stops at some point. When reviewing the issue with @jonespm, we suspected that there could be an issue with Playwright on my version of Python (3.12). Because the script runs for everyone else I don't think this should block this PR, but I'm just noting what I'm seeing in case someone encounters a similar problem with Playwright.

Reply:
Sounds good. Mine is still at Python 3.10.0 and I will hold on from upgrades.

Reply:
I'm able to reproduce this with Python 3.11.1. It seems to close after around 5 minutes. I thought this was something with the session token expiring but even trying to renew that it still just closes without any errors.

It's no longer running for me correctly to completion. How long does it take to complete for you @zqian. I'm trying to search through playwright issues to see if it's a something that's been reported and while I've seen some issues they're older and nothing that seems  to match this. 

I feel like the only thing I haven't tried is using a browser other than Chromium.